spec/index.bs

```diff
@@ -3693,6 +3696,10 @@ interface mixin GPUProgrammablePassEncoder {
     void pushDebugGroup(USVString groupLabel);
     void popDebugGroup();
     void insertDebugMarker(USVString markerLabel);
+
+    void beginPipelineStatisticsQuery(GPUQuerySet querySet, GPUSize32 queryIndex);
```

Comment:
can we limit this PR to timestamp queries only?

Reply:
I think this is just a merge error; beginPipelineStatisticsQuery is already in master.

Reply:
Yes, `beginPipelineStatisticsQuery` and `endPipelineStatisticsQuery` have been changed in #797, this two lines should be added when git rebase, I will remove them.

Reply:
Oh, I forgot that timestamp query is also not supported on render bundle encoder, as same as pipeline statistics query,  so we need to move `writeTimestamp` to render encoder and compute encoder.